helix-core/src/main/java/org/apache/helix/HelixCloudProperty.java

```diff
@@ -0,0 +1,150 @@
+package org.apache.helix;
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import java.io.InputStream;
+import java.util.Collections;
+import java.util.List;
+import java.util.Properties;
+import org.apache.helix.model.CloudConfig;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+/**
+ * hold helix cloud properties
+ */
+public class HelixCloudProperty {
+  private static final Logger logger = LoggerFactory.getLogger(HelixCloudProperty.class.getName());
+  private final String AZURE_CLOUD_PROPERTY_FILE = SystemPropertyKeys.AZURE_CLOUD_PROPERTIES;
+
+  private Boolean _isCloudEnabled;
+  private String _cloudId;
+  private String _cloudProvider;
+  private List<String> _cloudInfoSources;
+  private String _cloudInfoProcessorName;
+  private String _maxRetry;
+  private String _cloudConnectionTimeout;
+  private String _cloudRequestTimeout;
+  private Properties _customizedProperties = new Properties();
+
+  /**
+   * Initialize Helix Cloud Property based on the provider
+   * @param
+   */
+  public HelixCloudProperty(CloudConfig cloudConfig) {
+    this.setCloudEndabled(cloudConfig.isCloudEnabled());
+    this.setCloudId(cloudConfig.getCloudID());
+    this.setCloudProvider(cloudConfig.getCloudProvider());
+    switch (cloudConfig.getCloudProvider()) {
+      case "AZURE":
+        Properties azureProperties = new Properties();
+        try {
+          InputStream stream =
+              Thread.currentThread().getContextClassLoader().getResourceAsStream(AZURE_CLOUD_PROPERTY_FILE);
+          azureProperties.load(stream);
+        } catch (Exception e) {
+          String errMsg = "fail to open properties file: " + AZURE_CLOUD_PROPERTY_FILE;
+          throw new IllegalArgumentException(errMsg, e);
+        }
+        logger.info("load helix Azure cloud properties: " + azureProperties);
```

Comment:
Could you consider revising this to "Successfully loaded Helix Azure cloud properties"?